lib/logto_client.dart

```diff
@@ -11,13 +10,30 @@ import '/src/modules/pkce.dart';
 import '/src/modules/token_storage.dart';
 import '/src/utilities/utils.dart' as utils;
 import 'logto_core.dart' as logto_core;
-import 'logto_core.dart';
 
-export '/src/interfaces/logto_config.dart';
-export '/src/interfaces/openid.dart';
+export '/src/interfaces/logto_interfaces.dart';
 export '/src/utilities/constants.dart';
-
-// Logto SDK
+export '/src/exceptions/logto_auth_exceptions.dart';
+
+/**
+ * LogtoClient
+ * 
+ * The main class for the Logto SDK.
+ * It provides all the user authentication and authorization methods.
+ * 
+ * @param config: LogtoConfig - the basic configuration object for the Logto SDK.
+ * @param storageProvider: LogtoStorageStrategy (optional) - default is [InMemoryTokenStorage] used for storing tokens.
+ * @param httpClient: http.Client (optional) - custom [http.Client] to be used for making http requests.
+ * 
```

Comment:
just double check this is the standard dart comment block and can be read by IDEs

Reply:
yep multi-line comments